avocado/plugins/run.py

```diff
@@ -157,6 +157,13 @@ def configure(self, parser):
                                      'will be kept in the test suite found '
                                      'previously to filtering.'))
 
+        parser.add_argument("--log-test-data-directories", action="store_true",
+                            help=("Logs the possible data directories for each"
+                                  "test. This is helpful when writing new "
+                                  "tests and not being sure where to put data"
+                                  "files. Look for \"Test data "
+                                  "directories\" in your test log"))
```

Comment:
It'd be nice to keep `parser` arguments together. Would you please move it to line `123` to make it easier to find the arguments that belong together?